requirements.txt

```diff
@@ -1,11 +1,9 @@
 # This pulls in all the libraries needed to run Selenium tests
 # on Mozilla WebQA projects
 
```

Comment:
Nit: I think we can get rid of the comment in this file, I've been removing it from other projects.